_config.php

```diff
@@ -1,5 +0,0 @@
-<?php
-
-use SilverStripe\Dev\Deprecation;
-
-Deprecation::notification_version('4.0.0', 'tractorcow/silverstripe-fluent');
```

Comment:
Deprecation notices don't get filtered by version number anymore, so this isn't doing us any good being here.